www/source/blog/2018-06-13-base-plans-refresh.html.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Base Plans Refresh is Coming to Habitat Core Plans!
+date: 2018-06-13
+author: Nell Shamrell-Harrington
+tags: core plans
+category: update
+classes: body-article
+---
+
+Greetings, fellow Habicats!
+
+We will be refreshing the base plans of Core Plans on Tuesday, June 19th.  Let's go over what this means for us and, more important, what it means for you.
+
+## Base Plans 101
+
+Currently, there are about 500 Habitat plans under the Core Origin on Builder.
+
+These include everything from services like Postgresql to binaries like go to very low level system libraries, all of which you can easily pull into your own Habitat artifacts.
+
+### What's a Base Plan?
+
+The Base Plans are a group of plans (mostly low level systems libraries) that:
+* are used by nearly every other plan (or a dependency of that plan) on Builder
+* are used to build Habitat itself
+* need to be built in a certain order.
+
+You can see [the full list of Base Plans](https://github.com/habitat-sh/core-plans/blob/master/CODEOWNERS) in the Core Plans CODEOWNERS file.
```

Comment:
Once `fnichol/teh-futur` PR merges, the Base Plans are broken out into a dedicated file so that it's extremely obvious: https://github.com/habitat-sh/core-plans/blob/fnichol/teh-futur/base-plans.txt

Reply:
Cool - I'll update this when it merges.